README.md

```diff
@@ -1,2 +1,56 @@
-# diagrams
-Utility tool to generate mermaid diagrams for Warehouse.ai
+# `@wrhs/diagrams`
+
+[![Version npm](https://img.shields.io/npm/v/@wrhs/diagrams.svg?style=flat-square)](https://www.npmjs.com/package/diagrams)
+[![License](https://img.shields.io/npm/l/@wrhs/diagrams.svg?style=flat-square)](https://github.com/warehouseai/diagrams/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/@wrhs/diagrams.svg?style=flat-square)](https://npmcharts.com/compare/diagrams?minimal=true)
+[![Dependencies](https://img.shields.io/david/warehouseai/diagrams.svg?style=flat-square)](https://github.com/warehouseai/diagrams/blob/master/package.json)
+[![CircleCI](https://circleci.com/gh/warehouseai/diagrams.svg?style=svg)](https://circleci.com/gh/warehouseai/diagrams)
+
+Utility to generate consistent [Mermaid] diagrams for [Warehouse.ai] modules.
+
+## Install
+
+```sh
+npm install @wrhs/diagrams --save-dev
+```
+
+## Usage
+
+The easiest way to use this tool is by adding an `npm` command to the
+consuming module's `package.json`
+
+```json
+"scripts": {
+  "diagrams": "wrhs-diagrams --target=./docs --theme=forest"
```

Comment:
should that show a --source or --file? being diagrammed?

Reply:
Will at `--source`, although they have defaults. `--file` is not required per se.